about.html

```diff
@@ -0,0 +1,119 @@
+                <h3 class="h3-headline">"Hello cookie lovers!"</h3>
+                <h1>Cookie Makers<br>
+                    International Course in the Art of Cookie
+                    2023</h1>
+                <div class="line-bar"></div>
+                <div class="container-p-about">
+                    <p class="paragraphs">The international cookie masters course is made for those who are passionate about this
+                        wonderful art of pastry, but specifically about cookies. We all like cookies but not everyone knows how to do
+                        the best techniques to make the perfect cookies for our palate and in this course we take care of that.</p>
+                </div>
+                <p class="paragraphs p-about">Please contact us per Email for any further questions about CM International course 2023! <br>
+                    <a href="#">CMcourse@cookiemasters.org</a>
+                </p>
+            </section>
+        </main>
+    </div>
+
+    <div class="border">
+        <section class="logo-about-section margin-container">
+            <h2 class="black-title">Cookie Makers<br>
+                International Course 2023 Logo</h2>
+            <div class="line-bar"></div>
+            <p class="paragraphs">The Cookie Masters 2023 logo was designed in the process of creating this course by our
+                team on August 29, 2022.</p>
+            <div class="container-logo">
+                <img src="img/Cookie-Masters-2023-logo.png" alt="Cookie Masters logo">
+            </div>
+        </section>
+    </div>
+
+    <section class="past-course-section margin-container">
+        <h2 class="black-title">See the past International Courses</h2>
+        <div class="line-bar"></div>
+        <p class="paragraphs past-course-p">Go check out our past international courses.</p>
+        <div class="past-courses">
+            <div class="background-course bg-1">
+                <div class="background-course-text">
+                    <h3>2022</h3>
+                    <p>Cookie Masters Spain</p>
+                </div>
+            </div>
+            <div class="background-course bg-2">
+                <div class="background-course-text">
+                    <h3>2021</h3>
+                    <p>Cookie Masters Mexico</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <div class="partner-section">
+        <h2 class="title-white">Partner</h2>
+        <div class="line-bar"></div>
+        <div class="partner-branch">
+            <img src="img/Nestlé-Logo.png" alt="nestle logo">
+            <img src="img/Samsung-Logo.png" alt="samsung logo">
+            <img src="img/Spotify-Logo.png" alt="spotify logo">
+            <img src="img/Dunkin_Donuts-Logo.png" alt="dunkin donuts logo">
+            <img src="img/Tim_Hortons-Logo.png" alt="tim hortons logo">
+        </div>
+    </div>
+
+    <footer>
```

Comment:
- [ ] great job. please the  about page desktop size footer has a dark background. make this change to align with the original design. 
- [ ] kindly add a little space between the footer logo and text(do for home page too). nice work.
your design
![Screenshot (959)_LI](https://user-images.githubusercontent.com/65298048/232850072-a1b16042-0a26-4b8e-8059-2a27c44d2f19.jpg)
original design
![Screenshot (960)](https://user-images.githubusercontent.com/65298048/232850245-034499b3-4ac5-4477-b93b-bf91ca3d6606.png)